src/data/restore.ts

```diff
@@ -50,6 +50,7 @@ function migrateElementWithProperties<T extends ExcalidrawElement>(
     height: element.height || 0,
     seed: element.seed ?? 1,
     groupIds: element.groupIds || [],
+    strokeSharpness: element.strokeSharpness,
```

Comment:
This will not work since the default is `undefined` for old scenes. Here we can also set the default based on type.